arcade/examples/sprite_rotation.py

```diff
@@ -0,0 +1,76 @@
+"""
+Rotating Sprites Around Points
+
+A simple program to show how someone could implement sprites rotating around points.
+
+Artwork from https://kenney.nl
+"""
+import arcade
+
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+
+SCREEN_TITLE = "Rotating Sprite Example"
+
+
+class RotatingSprite(arcade.Sprite):
+
+    def rotate_around_point(self, point, degrees, change_angle=True):
+        """
+        rotates the sprite around a defined point by the set amount of degrees
+        """
+
+        if change_angle:
+            # This is so the direction the sprite faces changes when rotating.
+            # It isn't necessary to have this. For example, you would want a rotating platform to always face upwards.
+            self.angle += degrees
+
+        # use arcade.rotate_point to rotate the sprites center x and y by the point.
+        self.center_x, self.center_y = arcade.rotate_point(self.center_x, self.center_y, point[0], point[1], degrees)
+
+
+class ExampleWindow(arcade.Window):
+
+    def __init__(self):
+        super().__init__(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
+
+        # This first example is similar to the spinning fireballs from mario.
+        self.example_sprite_1 = RotatingSprite(":resources:images/space_shooter/laserBlue01.png",
```

Comment:
**Optional:** You could give the moving sprites more descriptive variable names. Perhaps something like `self.left_laser_bar` and `self.right_circling_platform`?